packages/main/src/WheelSlider.hbs

```diff
@@ -19,14 +19,25 @@
 	<div id="{{this._id}}--selection-frame" class="ui5-wheelslider-selection-frame"></div>
 		<div id="{{this._id}}--wrapper" class="ui5-wheelslider-wrapper">
 			{{#if expanded}}
-				<ul id="{{this._id}}--items-list">
+				<ul id="{{this._id}}--items-list" role="listbox" aria-labelledby="{{label}}">
```

Comment:
is this {{label}} an ID

Reply:
{{label}} is not an ID. I've mixed the two possible solutions.
I've changed to aria-label instead.